ckanext/scheming/helpers.py

```diff
@@ -27,6 +27,9 @@ def lang():
     from ckantoolkit import h
     return h.lang()
 
+@helper
+def scheming_composite_separator():
+    return config.get('scheming.composite.separator','|')
```

Comment:
Probably it would be better to keep this setting as the default '-' and override if needed.
Could this affect existing schemas? If so we should mode this to the scheming yaml / json instead providing a per type setting.

Reply:
good point about the default separator character. I have made that change. I don't think this will affect existing schemas, now that it is using the correct default separator char. A per field override is an interesting idea though. It would be a bit of work as everywhere the separator is used we would also need to check which field we are handling and check the schema. So far, my experience is that a global setting is meeting all our needs in cioos.